server/templates/details.html.tmpl

```diff
@@ -142,10 +147,22 @@
   </ul>
 {{end}}
 
-{{define "result-requires-details"}}
-  <b class="font-bold text-sm">This rule requires review from</b>
+{{define "result-patterns-details"}}
+  <b class="font-bold text-sm">This rule requires {{.Requires.Count}} review(s) using any of these patterns</b>
```

Comment:
Good call adding the count to all of these - I'm not sure why we didn't add it earlier. I think it would be better to only show the count once before all the other approval details. With the count repeated, people could interpret it as if they need N reviews for each section instead of N reviews total for the rule.

Reply:
yea that's fair, i was trying to play with the wording so that wouldn't be the case, but I think this is a better idea.